dist/src/main/resources/api/backup-management-api.yaml

```diff
@@ -88,8 +82,31 @@ paths:
         '502':
           description: Operation failed because connection to the broker or to the backup storage is lost.
           $ref: '#/components/responses/Error'
+    delete:
+      summary: Delete a backup
+      description: Delete a backup with the given id
+      parameters:
+        - $ref: '#/components/parameters/BackupId'
+      responses:
+        '204':
+          description: Backup is deleted
+        '500':
+          description: Internal error
+          $ref: '#/components/responses/Error'
+        '502':
+          description: Operation failed because connection to the broker or to the backup storage is lost.
+          $ref: '#/components/responses/Error'
```

Comment:
:thinking: I think what you want here is then `504`

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#server_error_responses 

> [504 Gateway Timeout](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504)
> This error response is given when the server is acting as a gateway and cannot get a response in time.
> 

Whereas 502 is:


> [502 Bad Gateway](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/502)
> This error response means that the server, while working as a gateway to get a response needed to handle the request, got an invalid response.
>

Reply:
Oh.. yes. `504` suits better in this case. Thanks for pointing this out. I will check with other teams.

Reply:
Will update it after syncing with other teams. For now, I will merge this.